spec/features/user_creates_review_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+feature "user creates review", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.